docs/source/troubleshooting/index.rst

```diff
@@ -101,16 +101,71 @@ If this happens, then you should follow these remediation steps:
+   $ curl -X POST --header 'content-type: application/json' '<product-base-url>/persistent-lock/release' -d '"427eb02a-f017-40cd-8d08-0a163315029a"'
+
+Clear with CQL (for Cassandra)
+------------------------------
+
+.. warning::
+
+   The Backup Lock is serialised differently than the Schema Mutation Lock. In particular, truncating the persisted
+   locks table will **not** release the Backup Lock, and will in fact put your cluster in a bad (though recoverable)
+   state!
+
+.. tip::
+
+   The steps below are Cassandra-specific, but the serialisation mechanics we use for other key-value services are very
+   similar. You will want to restore the relevant cell in your key-value service to the value documented below.
+
+If you are unable to find the currently-held lock in the logs, this approach may be helpful.
+The state of persisted locks is stored in the ``_persisted_locks`` table in your AtlasDB keyspace; specifically, the
+state of the backup lock is stored in a cell with row name ``BackupLock`` and column name ``lock``.
+
+.. code-block:: none
+
+   cqlsh> USE keyspace;
+   cqlsh:keyspace> SELECT * FROM "_persisted_locks";
+
+    key                    | column1    | column2 | value
+   ------------------------+------------+---------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    0x4261636b75704c6f636b | 0x6c6f636b |      -1 | 0x7b226c6f636b4e616d65223a224261636b75704c6f636b222c22696e7374616e63654964223a2234323765623032612d663031372d343063642d386430382d306131363333313530323961222c22726561736f6e223a22666f6f227d
+
+The ``value`` stored here is a serialised representation of the JSON ``LockEntry``; that included in the table above
+actually deserialises to
+
+.. code-block:: none
+
+   '{"lockName":"BackupLock","instanceId":"427eb02a-f017-40cd-8d08-0a163315029a","reason":"foo"}'
+
+AtlasDB interprets a specific ``LockEntry`` value as meaning that the lock is available:
+
+.. code-block:: java
+
+   // '{"lockName":"BackupLock","instanceId":"00000000-0000-0000-0000-000000000000","reason":"Available"}'
+   public static final LockEntry LOCK_OPEN = ImmutableLockEntry.builder()
+               .lockName(BACKUP_LOCK_NAME)
+               .instanceId(UUID.fromString("0-0-0-0-0"))
+               .reason("Available")
+               .build();
+
+Thus, we can set the relevant cell to be the serialised value of the backup lock.
+
+.. code-block:: none
+
+   cqlsh:keyspace> UPDATE "_persisted_locks" SET value=0x7b226c6f636b4e616d65223a224261636b75704c6f636b222c22696e7374616e63654964223a2230303030303030302d303030302d303030302d303030302d303030303030303030303030222c22726561736f6e223a22417661696c61626c65227d WHERE key=0x4261636b75704c6f636b AND column1=0x6c6f636b AND column2=-1 ;
```

Comment:
OK, fair enough - I didn't think it was necessary since the user should be sure that the process has died (warning box at beginning of section), but it probably wouldn't hurt to put another safeguard here. Added the if-clause.

(And yep, I tested the commands/workflow with a local AtlasDbEteServer and Cassandra instance.)